sw/device/lib/crypto/drivers/entropy.h

```diff
@@ -51,6 +51,16 @@ typedef struct entropy_seed_material {
   uint32_t data[kEntropySeedWords];
 } entropy_seed_material_t;
 
+/**
+ * Constant empty seed material for the entropy complex.
+ *
+ * This is convenient to have available for some implementations.
+ */
+static const entropy_seed_material_t kEntropyEmptySeed = {
```

Comment:
You probably need to move this to `entropy.c` and declare it here as an `extern` so that it is more stylistic.